apps/docs/content/frameworks/laravel/public-access.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: Public Access for Laravel Applications
+description: Learn how to configure public access for your Laravel application using Zerops subdomains or custom domains
+---
+
+import Image from '/src/components/Image';
+
+Firstly, make sure your zerops.yml is configured correctly with a readiness check and health check.
+
+```yaml
+zerops:
+  - setup: app
+    build:
+      base:
+        - php@8.4
+      os: alpine
+    deploy:
+      readinessCheck:
+        httpGet:
+          port: 80
+          path: /up
+    run:
+      base: php-nginx@8.4
+      os: alpine
+      healthCheck:
+        httpGet:
+          port: 80
+          path: /up
+```
+
+After your zerops.yml is configured correctly, you can enable public access for your application.
+
+
+## Public Access Options
+
+There are two ways to make your Laravel application publicly accessible on Zerops:
+1. Using a Zerops subdomain (yourapp.zerops.io)
```

Comment:
the example is a bit misleading, it's `.app` anyway, but I wouldn't specify it

Reply:
alright, now that i think about it - it's unnecessary to add an example